spec/system/provider_interface/provider_gives_feedback_for_rejected_by_default_spec.rb

```diff
@@ -73,41 +73,48 @@ def and_i_give_reasons_as_feedback_for_the_application
   end
 
   def and_i_check_the_feedback_given
-    expect(page).to have_link('Back', href: new_provider_interface_rejection_path(@application_choice))
-
     expect(page).to have_content('Check details and give feedback')
+    expect(page).to have_content('The candidate will be sent this email:')
+
+    email = page.all('.app-email-preview').first.text.split("\n")
 
-    rows = page.all('.govuk-summary-list__row')
+    expect(email[0..3]).to eq([
+      "Dear #{@application_choice.application_form.first_name},",
+      "Thank you for your application to study #{@application_choice.current_course_option.course.name_and_code} at #{@application_choice.current_course_option.provider.name}.",
+      'On this occasion, the provider is not offering you a place on this course.',
+      'They’ve given the following feedback to explain their decision:',
+    ])
 
-    expect(rows[0].text.split("\n")).to eq([
+    expect(email[4..7]).to eq([
```

Comment:
If the exact line numbers are not critical would it make sense to use `expect(email).to include([...])` instead? That way the test won't need changing if the other content changes.

Reply:
introducing tech debt but i'm gonna add a card to refactor this all to move to a separate component and/or find a way to reuse the existing mailers